pkg/convert/convert.go

```diff
@@ -88,10 +67,24 @@ func NewCmdConvert(setupCtx context.Context, ioStreams genericclioptions.IOStrea
 	o := NewOptions(ioStreams)
 
 	cmd := &cobra.Command{
-		Use:                   "convert",
-		Short:                 "Convert cert-manager config files between different API versions",
-		Long:                  longDesc,
-		Example:               example,
+		Use:   "convert",
+		Short: "Convert cert-manager config files between different API versions",
+		Long: templates.LongDesc(`
```

Comment:
Does this have to do with "making the binary dynamically determine whether it is a kubectl plugin"? 

Inlining these `Long` paragraphs is a good idea, but it makes it extra hard to review the actual changes.

Reply:
Yes, I inlined these paragraphs because sometimes we call `build.WithTemplate(setupCtx,` which takes the `setupCtx` argument to determine if it is running as `cmctl` or `kubectl cert-manager`.